pkg/registry/apis/provisioning/register.go

```diff
@@ -70,6 +70,7 @@ var (
 type APIBuilder struct {
 	urlProvider      func(namespace string) string
 	webhookSecretKey string
+	isPublic         bool
```

Comment:
isPublic does not feel right -- the whole API is not public

IIUC, this is saying "we think this is hosed on on a non-localhost domain"